src/wasm-js.cpp

```diff
@@ -250,10 +250,12 @@ extern "C" void EMSCRIPTEN_KEEPALIVE instantiate() {
       switch (type) {
         case none: return Literal();
         case i32: return Literal((int32_t)ret);
+        case i64: WASM_UNREACHABLE();
```

Comment:
Seems inconsistent to have `i64` abort, but `unreachable` breaks out of the switch, to abort later.